ui/pages/confirmations/components/confirm/info/shared/send-heading/send-heading.tsx

```diff
@@ -75,11 +78,12 @@ const SendHeading = () => {
       >{`${displayTransferValue} ${tokenSymbol}`}</Text>
     );
 
-  const TokenFiatValue = fiatDisplayValue && (
-    <Text variant={TextVariant.bodyMd} color={TextColor.textAlternative}>
-      {fiatDisplayValue}
-    </Text>
-  );
+  const TokenFiatValue = Boolean(fiatDisplayValue) &&
```

Comment:
As we duplicate this and the hook usage in three places, is it worth a `FiatValue` component that encapsulates this?

Reply:
I think this is only used in two places, and since calculations for fiatDisplayValue are already encapsulated in the `useTokenValues` hook, creating a separate component seems a bit premature or at least optional at this point.

The parent SendHeading and NativeSendHeading components would still rely on the same useTokenValues if we created a FiatDisplay component, so it doesn't seem like a big gain in modularity.